src/mlpack/tests/distribution_test.cpp

```diff
@@ -188,6 +189,50 @@ BOOST_AUTO_TEST_CASE(MultiDiscreteDistributionTrainProTest)
   BOOST_REQUIRE_CLOSE(d.Probability("2 1 0"), 0.015625, 1e-5);
 }
 
+/**
+ * Test the LogProbability() function, for multiple points in the multivariate
+ * Discrete case.
+ */
+BOOST_AUTO_TEST_CASE(DiscreteLogProbabilityTest)
+{
+  // Same case as before.
+  DiscreteDistribution d("5 5");
+
+  arma::mat obs("0 2;"
+                "1 2;");
+
+  arma::vec logProb;
+
+  d.LogProbability(obs, logProb);
+
+  BOOST_REQUIRE_EQUAL(logProb.n_elem, 2);
+
+  BOOST_REQUIRE_CLOSE(logProb(0), -3.2188758248682, 1e-3);
+  BOOST_REQUIRE_CLOSE(logProb(1), -3.2188758248682, 1e-3);
+}
+
+/**
+ * Test the Probability() function, for multiple points in the multivariate
+ * Discrete case.
+ */
+BOOST_AUTO_TEST_CASE(DiscreteProbabilityTest)
+{
+  // Same case as before.
+  DiscreteDistribution d("5 5");
+
+  arma::mat obs("0 2;"
+                "1 2;");
+
+  arma::vec Prob;
+
+  d.Probability(obs, Prob);
+
+  BOOST_REQUIRE_EQUAL(Prob.n_elem, 2);
+
+  BOOST_REQUIRE_CLOSE(Prob(0), 0.4000000000000, 1e-3);
+  BOOST_REQUIRE_CLOSE(Prob(1), 0.4000000000000, 1e-3);
```

Comment:
Ah, sorry, I approved this but missed a couple little things---`Prob` should be `prob` (style) and I think that the probability of each observation should be `0.2 * 0.2 = 0.04`, not `0.4`.  If you fix that the build should pass.

Reply:
Yeah, I saw that probability error. I will rectify it asap.